jsonnet/main.jsonnet

```diff
@@ -26,9 +26,11 @@ local kp = (import 'kube-prometheus/kube-prometheus.libsonnet') +
            (import 'kube-state-metrics.jsonnet') +
            (import 'grafana.jsonnet') +
            (import 'alertmanager.jsonnet') +
-           (import 'prometheus.jsonnet') + {
+           (import 'prometheus.jsonnet') +
+           (import 'cluster-monitoring-operator.jsonnet') + {
   _config+:: {
     namespace: 'openshift-monitoring',
+    name: 'cluster-monitoring-operator',
```

Comment:
This is the name of the project. I am open to suggestions to renaming the field

Reply:
we should nest this in a cluster-monitoring-operator config